### DISCUSSION TITLE
How do I customize the grid prefix

### DISCUSSION THREAD

**Discussion by a42356** [tailwindlabs/tailwindcss#1466] [General]
autoprefixer bad
How to customize GIRD in Tailwindcss

```
grid {
    display: grid;
  }
```

to

```
.grid {
  display: -ms-grid;
  display: grid;
}
```

**Comment by adamwathan** [comment]
Use autoprefixer. If you need to support IE11 though I would encourage you not to bother trying to use grid, the IE11 implementation is unusable.
